script/build-linux

```diff
@@ -5,7 +5,7 @@ set -ex
 ./script/clean
 
 TAG="docker-compose"
-docker build -t "$TAG" .
+docker build -t "$TAG" . | tail -n 200
```

Comment:
Ya, I made this change because on travis it causes 9000+ lines of debug output, which causes the log buffer to fill and requires us to download the log instead of being able to use the UI.  We could fix it by adding `> /dev/null` to some of the run lines in the `Dockerfile`, but then we'd lose it in all cases (and that's just a lot more work to track down).

On jenkins the image is already build and this line only hits the cache, so if there were issues we'd already see them the first time it runs the build.  If there are critical failures we should see them in the last 200 lines.